index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="navbar">
+        <ul>
+             <li><a href="#about">About</a></li>
+             <li><a href="#projects">Work</a></li>
+             <li><a href="#contact">Contact</a></li>
+            
+        </ul>
+    </div>
+
+    <div id="about">
+        <h1>Hey!this is Bernad and Oliver.</h1>
+        <h3>We are software developers.</h3>
+    </div>
+    <h1 style="color:rgb(214, 12, 123)">These are some of our projects</h1>
+    <div id="projects" class="projects">
+        
+        <div class="flex project1">
+            <h3 class="project-title">Entertainment</h3>
+            <img src="hot.png" height="300px" width="300px" alt="project1">
+            <a href="#"><button>View</button></a>
+             
+        </div>
+        <div class="flex project2">
+            <h3 class="project-title">Please sign up</h3>
+            <img src="copy1.png" height="300px" width="300px" alt="project1">
+            <a href="#"><button>View</button></a>
+             
+        </div>
+        <div class="flex project3">
+            <h3 class="project-title">Calculator</h3>
+            <img src="sample1.png" height="300px" width="300px" alt="project1">
+            <a href="#"><button>View</button></a>
+             
+        </div>
+    </div>
+    <div class="flex project3">
+        <h3 class="project-title">Memorable Quotes</h3>
+        <img src="sample2.png" height="300px" width="300px" alt="project1">
+        <a href="#"><button>View</button></a>
+    </div>
+    
+
+    </div>
```

Comment:
Please consider using a section instead of a div for the projects section for semantic purposes.  A section is used to group a collection of related items.